packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -380,13 +380,17 @@ export class DeltaManager
      * about current or last connection (if there is no connection at the moment)
     */
     public connectionProps(): ITelemetryProperties {
+        const common = {
+            connectionMode: this.connectionMode,
+        };
         if (this.connection !== undefined) {
             return {
-                sequenceNumber: this.lastSequenceNumber,
+                ...common,
                 connectionMode: this.connectionMode,
```

Comment:
`connectionMode` is duplicated in `common`?  I think you intent to keep `sequenceNumber` here?

Reply:
Yes, wrong thing copied over - thanks!